src/components/aboutUs/components/TeamProfile.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Avatar, Card, Icon, List } from 'antd';
+
+import { ABOUT_US, SOCIAL_MEDIA, TEAM_INFO } from '../constants';
+
+const { Meta } = Card;
+const { 
+    CLASSNAMES: { AVATAR, TEAM_CARD },
+    STRINGS: { SQUARE }, 
+} = ABOUT_US;
+
+const actionList = SOCIAL_MEDIA.map(social => (<Icon type={social} key={social} />));
+
+class TeamProfile extends React.Component {
+    render() {
+        return(
+            <List  
+                bordered={false}
+                dataSource={TEAM_INFO}
+                grid={{ gutter: 12, lg: 4, md: 1.7, sm: 1.8, xs: 1.7, xxl: 3 }}
+                renderItem={item => (
+                    <List.Item>   
+                        <Card
+                            key={item.name}
+                            hoverable
+                            bordered={false}
+                            className={TEAM_CARD}
+                            cover={
+                                <Avatar src={item.photo} shape={SQUARE} className={AVATAR} />
+                            }
+                            actions={actionList}
+                        >
+                            <Meta
+                                title={item.name}
+                                description={item.description }
+                            />
+                        </Card> 
+                    </List.Item>
+                )}
+            />
+        );
+    }
+}
+
+export default TeamProfile;
```

Comment:
@Way2nnadi Is it OK to have two `export default` s in a module like this?

Reply:
you can have multiples `export` s but only one `export default`